atlasdb-api/src/main/java/com/palantir/atlasdb/metrics/MetricPublicationArbiter.java

```diff
@@ -35,10 +38,10 @@
 public class MetricPublicationArbiter implements Predicate<MetricName> {
     private static final Logger log = LoggerFactory.getLogger(MetricPublicationArbiter.class);
 
-    private final Map<MetricName, List<MetricPublicationFilter>> singleMetricFilters;
+    private final Map<MetricName, Set<DeduplicatingFilterHolder>> singleMetricFilters;
 
     public MetricPublicationArbiter(
-            Map<MetricName, List<MetricPublicationFilter>> singleMetricFilters) {
+            Map<MetricName, Set<DeduplicatingFilterHolder>> singleMetricFilters) {
```

Comment:
I'll reshuffle things around a bit - this is a bit strange to keep on the API because `DedupFilterHolder` is private